src/test/java/com/google/api/generator/test/framework/Differ.java

```diff
@@ -50,22 +50,28 @@
     try {
       diff = DiffUtils.diff(original, revised);
     } catch (DiffException e) {
-      throw new ComputeDiffException("Could not compute the differences.");
+      throw new ComputeDiffException("Could not compute the differences.", e);
     }
     List<String> unifiedDiff =
         UnifiedDiffUtils.generateUnifiedDiff("golden", "codegen", original, diff, 2);
     return unifiedDiff;
   }
 
   private static class GoldenFileReadException extends RuntimeException {
-    public GoldenFileReadException(String errorMessage) {
-      super(errorMessage);
+
+    private static final long serialVersionUID = 7423787084310530945L;
```

Comment:
Would this be used somewhere? Same below.

Reply:
This is technically a best-practice requirement for all `Serializable` classes (which all `Exceptions` are), a bit dogmatic, agree.

Reply:
SG, thanks.